mappings/net/minecraft/world/dimension/PortalForcer.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_467 net/minecraft/world/dimension/PortalForcer
```

Comment:
I despise this name :p But seeing as its both Yarn and Notch name, I cant do much except to cry about it 😭